.eslintrc

```diff
@@ -1,7 +1,18 @@
+extends:
+  - plugin:fxa/server
+  - plugin:node/recommended
+
 plugins:
   - fxa
-extends: plugin:fxa/server
+  - node
+
+root: true
 
 rules:
-  handle-callback-err: 0
-  strict: 2
+  node/no-missing-require: off
+  node/no-unpublished-require: off
+  node/shebang: off
```

Comment:
same as comment above, let's make changes in eslint-plugin-fxa, land those as a pull request, then change this Git SHA in this pull request: https://github.com/mozilla/fxa-auth-server/blob/master/npm-shrinkwrap.json#L338